pages/press/index.js

```diff
@@ -0,0 +1,348 @@
+          <ul className="list-disc list-inside ml-4 my-4">
+            <li className="py-1">
+              <Paragraph typeStyle="body-md" className="my-4">
+                <b>Innovative</b>. There is no other specification that covers the messaging needs in the event-driven architecture that AsyncAPI is covering. 
+                What it tries to do is to integrate with the existing tools and remove walls for communication.
+              </Paragraph>
+            </li>
+            <li className="py-1">
+              <Paragraph typeStyle="body-md" className="my-4">
+              <b>Free</b>. It’s a free software project: it seeks the user's liberty by offering a tool that can be used and enhanced without restrictions.
+              </Paragraph>
+            </li>
+            <li className="py-1">
+              <Paragraph typeStyle="body-md" className="my-4">
+              <b>Open</b>. As it's an open-source project and not a company or a product, it works in a change of paradigm: it does not sell anything, it moves away from fierce competition and bets for cooperation.
+              </Paragraph>
+            </li>
+            <li className="py-1">
+              <Paragraph typeStyle="body-md" className="my-4">
+                <b>Multicultural and inclusive</b>. It’s an initiative with an international and multicultural vision. 
+                It’s organic, always in constant mutation and change. 
+                It also actively demands compliance with human rights, respect for racial, sexual, and cultural diversity equality. 
+                And, of course, the equality between women and men.
+              </Paragraph>
+            </li>
+            <li className="py-1">
+              <Paragraph typeStyle="body-md" className="my-4">
+                <b>Transparent</b>. The initiative it’s built on transparency, being one of the main values of AsyncAPI, both internally and externally including its financing. 
+                All the relevant data of the project is public. 
+                The tasks that are being and will be worked on are public and participatory through Slack and GitHub.
+              </Paragraph>
+            </li>
+            <li className="py-1">
+              <Paragraph typeStyle="body-md" className="my-4">
+                <b>Horizontal and participative</b>. Its structure is based on horizontality; most of the decisions are defined as a group, including the Community. 
+                Anyone can submit improvements, lines of action, or elements to be taken into account to strengthen AsyncAPI
+              </Paragraph>
+            </li>
+            <li className="py-1">
+              <Paragraph typeStyle="body-md" className="my-4">
+                <b>User-friendly</b>. AsyncAPI bets for a seamless API development experience. Users should be able to create their first API within minutes without prior knowledge of AsyncAPI. It has to be natural, simple, intuitive.
```

Comment:
```suggestion
                <b>User-friendly</b>. AsyncAPI bets for a seamless API development experience. Users should be able to create their first API within minutes without prior knowledge of AsyncAPI. It has to be natural, simple, and intuitive.
```

_"create their first API within minutes without prior knowledge of AsyncAPI"_

Honestly, this reads like a stretch. I would change this part.